<ui version="4.0" >
 <class>OpenLabUi</class>
 <widget class="QDialog" name="OpenLabUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Visual Netkit - Open Laboratory</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Laboratory path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="labPathLineEdit" />
     </item>
     <item>
      <widget class="QToolButton" name="browseButton" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="text" >
        <string>Browse...</string>
       </property>
       <property name="icon" >
        <iconset resource="../resources/VisualNetkit.qrc" >
         <normaloff>:/small/folder_vm</normaloff>:/small/folder_vm</iconset>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="stepsGroupBox" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="title" >
      <string>Open steps</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="checkBox_0" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Validating Laboratory</string>
        </property>
        <property name="tristate" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_1" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Creating Laboratory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Creating Virtual Machines</string>
        </property>
        <property name="tristate" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Creating Collision Domains</string>
        </property>
        <property name="tristate" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_4" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Creating Hardware Interfaces</string>
        </property>
        <property name="tristate" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_5" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Creating graphics elements</string>
        </property>
        <property name="tristate" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_6" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Connecting plugins</string>
        </property>
        <property name="tristate" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_7" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Drawing Network Graph</string>
        </property>
        <property name="tristate" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_8" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>All done</string>
        </property>
        <property name="tristate" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/VisualNetkit.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenLabUi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
